src/diffusers/models/attention.py

```diff
@@ -395,8 +449,27 @@ def __init__(self, embedding_dim, num_embeddings):
         self.linear = nn.Linear(embedding_dim, embedding_dim * 2)
         self.norm = nn.LayerNorm(embedding_dim, elementwise_affine=False)
 
-    def forward(self, x, timestep):
+    def forward(self, timestep):
         emb = self.linear(self.silu(self.emb(timestep)))
         scale, shift = torch.chunk(emb, 2)
-        x = self.norm(x) * (1 + scale) + shift
```

Comment:
This breaks previous models using this class no? `self.norm(...)` is now not used anymore I think

Reply:
it is used by calling it from outside this class for instances of where it is used... so nothing should be broken